superset-frontend/src/views/components/Menu.tsx

```diff
@@ -259,6 +280,15 @@ export function Menu({
           .ant-menu-submenu.ant-menu-submenu-popup.ant-menu.ant-menu-light {
             border-radius: 0px;
           }
+          .ant-menu-vertical
+            > .ant-menu-submenu.data-menu
+            > .ant-menu-submenu-title {
+            height: 28px;
```

Comment:
do we still use gridUnits?

Reply:
+1 here's and example:
https://github.com/apache/superset/blob/aa747219ad5fe1dbbf8090f937cb2299575047f8/superset-frontend/src/views/CRUD/data/database/DatabaseModal/styles.ts#L35

Reply:
This css style method can only grab the theme if it's passed in or being used directly by the css prop. I could do that and add more code. I think this would be a bit cleaner.

Reply:
Yeah, I think that at this point it would make sense to move all these styles to a new StyledGlobal component instead of keeping them inline.